### PR TITLE
Modified to support selective processing of services.

### PR DESCRIPTION
**Pull request by steingod**
Added command line options to decide whether services are processed or not and if WMS is present whether or not to include WMS-layers.

### CODE REVIEW

File: mmd_utils/nc_to_mmd.py [hunk 1]
```diff
@@ -519,5 +521,5 @@ def main(input_file=None, output_path='./',parse_services=False):
         output_name = 'multisensor_sic.xml'
         input_file = ('https://thredds.met.no/thredds/dodsC/sea_ice/'
                       'SIW-METNO-ARC-SEAICE_HR-OBS/ice_conc_svalbard_aggregated')
-    md = Nc_to_mmd(output_path, output_name, input_file, parse_services, True)
+    md = Nc_to_mmd(output_path, output_name, input_file, parse_services, parse_wmslayers, True)
```
Comment (mortenwh) [inline mmd_utils/nc_to_mmd.py:524]:
```suggestion
    md = Nc_to_mmd(output_path, output_name, input_file, parse_services, parse_wmslayers, print_file)
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
It looks ok. I think I found one minor bug, though.. We need to keep track that the same functionality is implemented in py-mmd-tools. I've added a comment to https://github.com/metno/py-mmd-tools/pull/39. When [that PR](https://github.com/metno/py-mmd-tools/pull/39) is done, I guess we can use the new module for this?

File: mmd_utils/nc_to_mmd.py (hunk 1 above)
Comment (steingod) [reply mmd_utils/nc_to_mmd.py]: is the patch added ok? Can we proceed on this?

Comment (mortenwh) [reply mmd_utils/nc_to_mmd.py]: it looks good to me

**Review by mortenwh — APPROVED** [review]

**Comment by steingod** [comment]
@mortenwh do you agree that we merge this? although @hevgyrt hasn't responded yet?

**Comment by mortenwh** [comment]
yes, go ahead :)

**Review by hevgyrt — APPROVED** [review]
Looks good to me

**Pull request merged**